File: source/UIPerf/app/src/main/java/com/example/uiperf/MyApp.java
package com.example.uiperf;

import android.app.Application;

import com.example.uiperflib.UIPerfMonitor;

public class MyApp extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        UIPerfMonitor.getInstance().startMonitor();
    }
}

File: source/KotlinStudy/wanandroiddemo/src/main/java/com/zero/wanandroiddemo/MyApp.kt
package com.zero.wanandroiddemo

import android.app.Application
import com.zero.wanandroiddemo.network.ApiClient

//open class APPLibray{
//    var innerDB:InnerDB? = null
//    get() = field!!
//    set(value) {
//        if(value!=null)
//         field = value
//    }
//    interface InnerDB{}
//}
class MyApp : Application(){

    override fun onCreate() {
        super.onCreate()
        //初始化了这个WanAndroidApi 实例对象
        ApiClient.instance.createRetrofit()
    }

}